code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -539,6 +539,16 @@
 	damage = 0
 	stamina = 40
 
+//pirate version
+/obj/item/gun/magic/hook/pirate
+	name = "boarding hook"
+	desc = "A long metal chain, with a large hook connected to the end. Once used to board ancient Merchant vessels, now used to drag un-suspecting Planet-crabs closer to you."
```

Comment:
i suspect "planet-crabs" might be a direct translation or a modified idiom that didn't quite make it to english very well.  You might want to make the description a little more clearly descriptive. Suggested desc below.

desc = "A long metal chain, with a heavy hook connected to the end that can make deadly blows in close quarters. Once used to board ancient merchant vessels, now used to drag unsuspecting victims into close range."